TO BE MERGED FOR 2.8.2 - MTV-2332: Update MTV recommendations for migrating VMS with large disks (#671)
kubev2v/forklift-documentation: documentation/modules/vmware-prerequisites.adoc

[IMPORTANT]
====

> In case of a power outage, data might be lost for a VM with disabled hibernation. However, if hibernation is not disabled, migration will fail.

Author: ```suggestion
In the event of a power outage, data might be lost for a VM with disabled hibernation. However, if hibernation is not disabled, migration will fail.
```

> In the event of a power outage, data might be lost for a VM with disabled hibernation. However, if hibernation is not disabled, migration will fail.

Author: ```suggestion
In the event of a power outage, data might be lost for a VM with disabled hibernation. However, if hibernation is not disabled, migration will fail
```

2+|`Cryptographic` privileges:
|`Cryptographic.Decrypt` |Allows decryption of an encrypted virtual machine.
|`Cryptographic.Direct access` |Allows access to encrypted resources.

> |===

Author: ```suggestion
|===
```

|`Cryptographic.Decrypt` |Allows decryption of an encrypted virtual machine.
|`Cryptographic.Direct access` |Allows access to encrypted resources.
|===

> 
> [TIP]
> ==== 
> Create a role in VMware with the permissions described in the preceding table and then apply this role to the *Inventory* section, as described in xref:creating-vmware-role-mtv-permissions_mtv[Creating a VMware role to apply MTV permissions]
> ====

Author: ```suggestion

```
